tests/coderabbit_eval/scripts/score.py

```
@@ -0,0 +1,249 @@
+        with open(path, encoding="utf-8") as handle:
+            payload = json.load(handle)
+    except OSError as exc:
+        raise ScoreError(f"cannot read labels file {path}: {exc}") from exc
+    except json.JSONDecodeError as exc:
+        raise ScoreError(f"invalid JSON in labels file {path}: {exc}") from exc
+
+    if not isinstance(payload, list) or not payload:
+        raise ScoreError(f"labels file {path} must be a non-empty JSON array")
+    for entry in payload:
+        if not isinstance(entry, dict) or "file" not in entry:
+            raise ScoreError(f"labels file {path} has an entry without a 'file' key")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)